packages/components/Notification/src/Notification.types.ts

```diff
@@ -39,6 +40,8 @@ export interface NotificationSlotProps {
   title?: ITextProps;
   message: ITextProps;
   action?: ButtonProps;
+  actionText?: ITextProps;
+  dismissIcon?: SvgProps;
```

Comment:
I think actionText and action are Redundant now?
Also, could we keep the naming the same? so `actionIcon` maybe?

Reply:
Also, could it be `IconProps` instead of `SvgProps` so that the developer can pass any type of icon we support in there?

Reply:
^ That suggestion also means some refactoring of ` getDismissIconPath` too